### ISSUE TITLE
Can't merge INV & BND calls

### ISSUE THREAD

**Issue by berguner** [dellytools/delly#89]
Hello,

I am trying to merge INV & BND type variants by using 

```
delly merge -b 250 -o merged.BND.bcf -t BND *.BND.bcf
delly merge -b 250 -o merged.INV.bcf -t INV *.INV.bcf
```
but I am getting this error;
```
terminate called after throwing an instance of 'std::runtime_error'
  what():  locale::facet::_S_create_c_locale name not valid
Aborted
```
Can you please check?

Thanks,
Bekir

**Comment by tobiasrausch** [comment]
Hi Bekir,

That's a strange error, I haven't seen this previously. Apparently something is wrong with the locale.
Can you try first setting LC_ALL and then rerunning the merge?

export LC_ALL=C

Does this work?

Thanks, Tobias

**Comment by berguner** [comment]
Hello Tobias,

I don't have this problem with the current version of delly. Thank you for the update.

Best,
Bekir

**Comment by zhujack** [comment]
Hello,

I got the following with delly/0.7.9, but not delly/0.7.8:

$ module load delly/0.7.9
$ delly merge -o sv.bcf sv/*.sv.bcf

terminate called after throwing an instance of 'std::runtime_error'
  what():  locale::facet::_S_create_c_locale name not valid
Aborted

Could you look into this?

Thanks,

Jack

**Comment by tobiasrausch** [comment]
Hi Jack,

I had this issue once on a system where I linked against an older Boost
version with Boost <v1.56.0. Unfortunately, previous Delly versions do not
show the linked HTSlib and Boost versions. I have just committed a change
for that so the next release will show with

delly --version

also the linked versions of Boost and HTSlib to debug these kind of errors
more easily. What worked for me at that time is resetting the locale before
running delly merge.

export LC_ALL=C; unset LANGUAGE

I am not sure if that will work for you but delly v0.7.8 should also be
fine for merging the Delly SV VCFs.

Best, Tobias
